package.json

```diff
@@ -38,7 +38,11 @@
     "webpack-dev-server": "^1.8.2"
   },
   "dependencies": {
-    "invariant": "^2.1.0",
-    "warning": "^2.0.0"
+    "invariant": "^2.1.0"
+  },
+  "peerDependencies": {
+    "react": "*",
```

Comment:
That's one possibility. Personally, I'd drop the `peerDependencies` entirely. You literally can't use this package without having `react` and `react-router` installed, and you can't do anything useful with it unless you already have `react-relay` installed either.

Technically this doesn't directly depend on anything in React 14. Arguably we should specify a React Router 1.0.0-beta dependency, but currently Relay is incompatible with v0.13.3 because of Relay's requirement for React 14, so in practice we don't get a lot out of explicitly specifying the peer dependency right now.